core/src/components/input/input.ionic.scss

```diff
@@ -19,10 +19,46 @@
 // Ionic Input Sizes
 // --------------------------------------------------
 
+:host(.input-size-medium) .native-wrapper {
+  min-height: 40px;
+}
+
 :host(.input-size-large) .native-wrapper {
   min-height: 48px;
 }
 
+// Target area
+// --------------------------------------------------
+:host .native-wrapper::after {
```

Comment:
~ No changes needed here. We should consider a mixin for these common styles that we are implementing for target area in multiple components.

Reply:
I'll add it to my todo list.